charts/mothership/templates/node.yaml

```diff
@@ -90,6 +90,23 @@ spec:
               name: node-volume
               readOnly: true
         {{- end }}
+        {{- if not $isSequencer }}
+        - name: generate-static-peer
+          image: curlimages/curl
+          command: ["sh", "-c"]
+          args:
+            - |
+              apk add jq ; \
+              curl -X POST -H "Content-Type: application/json" --data \
+              '{"jsonrpc":"2.0","method":"opp2p_self","params":[],"id":1}'  \
+              {{ $.Values.sequencer.gethRpc }} | jq -r ".peerID" > /node/bootnode ;
```

Comment:
maybe nodeRpc instead of gethRpc to use opp2p prefixed RPCs? (And I think it should be blocked from external network somehow)

might be better to use secrets

Reply:
> might be better to use secrets

is it about bootnode ENR or nodeRpc address itself??

Reply:
I think calling nodeRpc wouldn't be needed here if we preserve ENR itself into secrets

Reply:
i've restart one of the nodes and i checked ENR can be modified. i think our `--p2p.advertise-ip` could be resolved to different IP address because our load balancer has 3 public IPs.
so i think sticking our ENR to secret might not work well